modules/users/default.nix

```
@@ -182,7 +184,7 @@ in
         if [ -n "$u" ]; then
           if [ "$u" -gt 501 ]; then
             echo "deleting user ${name}..." >&2
-            dscl . -delete '/Users/${name}' 2> /dev/null
+            sysadminctl -deleteUser '${name}' 2> /dev/null
```

```suggestion
            sysadminctl -deleteUser '${name}'
```

Need to check if the `2> /dev/null` is still necessary

---

I'll keep it as `sysadminctl` outputs a lot of unnecessary stuff to stderr